packages/slate-editor/src/extensions/mentions/components/MentionsDropdown.tsx

```diff
@@ -37,36 +52,50 @@ export const MentionsDropdown = <V extends object>({
         ensureChildInView(suggestionsPanelRef.current, index, AUTO_SCROLL_SAFETY_MARGIN);
     }, [index, suggestionsPanelRef]);
 
-    useEffect(() => {
-        if (target && options.length > 0 && suggestionsPanelRef.current) {
-            const domRange = EditorCommands.toDomRange(editor, target);
-
-            if (!domRange) {
-                return;
-            }
-
-            const rect = RangeFix.getBoundingClientRect(domRange);
-
-            if (!rect) {
-                return;
-            }
-
-            const { left, height, top } = rect;
-            suggestionsPanelRef.current.style.top = `${top + window.pageYOffset + height}px`;
-            suggestionsPanelRef.current.style.left = `${left + window.pageXOffset}px`;
```

Comment:
This implementation based on `position: fixed` has stopped working after we've introduced a container element with `translate: 0 0` style (used by the new main navigation sidebar). This changes the containing block of the fixed-positioned element to the one with `translate: 0 0`, making the positioning not work anymore.

See https://developer.mozilla.org/en-US/docs/Web/CSS/position#fixed_positioning